nx-X11/config/cf/linux.cf

```diff
@@ -776,12 +776,13 @@ XCOMM binutils:	(LinuxBinUtilsMajorVersion)
 # endif
 # define LinuxMachineDefines	-D__sparc__
 # define ServerOSDefines	XFree86ServerOSDefines -DDDXTIME
-# define ServerExtraDefines	-DGCCUSESGAS XFree86ServerDefines
 # define AsVISOption		-Av9a
 # ifdef Sparc64Architecture
 #  define AsOutputArchSize	64
+#  define ServerExtraDefines	-DGCCUSESGAS XFree86ServerDefines -D_XSERVER64
```

Comment:
You could have probably used
```Imakefile
# define SparcCommonServerExtraDefines -DGCCUSESGAS XFree86ServerDefines
```

and later
```Imakefile
#  define ServerExtraDefines	SparcCommonServerExtraDefines -D_XSERVER64
[...]
#  define ServerExtraDefines	SparcCommonServerExtraDefines
```

but it probably doesn't matter a whole lot.